submissions/Oleksii_Drohachov/OOP-exercise/index.js

```diff
@@ -0,0 +1,116 @@
+class Creature {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.propsArray = ["species", "name", "gender", "saying", "friends"];
+  }
+  output = () => {
+    this.propsArray.length === 6
+      ? (this.prettyOutput = [
+          "name",
+          "species",
+          "gender",
+          "legs",
+          "saying",
+          "friends",
+        ])
+      : this.propsArray.length === 7
+      ? (this.prettyOutput = [
+          "name",
+          "species",
+          "gender",
+          "legs",
+          "hands",
+          "saying",
+          "friends",
+        ])
+      : (this.prettyOutput = this.propsArray);
```

Comment:
Nested ternaries considered a bad practice. Can we make this fragment more obvious? KISS